### PR TITLE
Update input data consistent with P6

### LINKED ISSUE #293: "inst_height_lowest" field can have negative values exported to ICE when running coupled.
## Description

The ATM ICs currently used in the coupled model can result in negative values of the field ``inst_height_lowest`` to be exported to the ICE. If these values get mapped to an ICE model point, the ICE model seg faults because it uses the ``inst_height_lowest`` value to calculate ``log(zlvl/zref)`` where ``zlvl`` is the imported height and ``zref=10m``.

## Additional context

This bug is IC dependent. It arose during testing for ufs-weather PR [#537](https://github.com/ufs-community/ufs-weather-model/pull/537). More information and testing can be found in this [thread](https://github.com/ufs-community/ufs-weather-model/issues/388#issuecomment-828559457).

## Output

When compile in debug mode, the following error is produced:

```
forrtl: error (65): floating invalid
Image              PC                Routine            Line        Source
fv3.exe            000000000EFE713E  Unknown               Unknown  Unknown
fv3.exe            000000000F1236C0  Unknown               Unknown  Unknown
fv3.exe            000000000889990B  Unknown               Unknown  Unknown
fv3.exe            000000000E323563  icepack_atmo_mp_a         250  icepack_atmo.F90
fv3.exe            000000000E328B16  icepack_atmo_mp_i         953  icepack_atmo.F90
fv3.exe            000000000E4ADF4F  icepack_therm_ver        2528  icepack_therm_vertical.F90
fv3.exe            000000000E204622  ice_step_mod_mp_s         316  ice_step_mod.F90
fv3.exe            000000000D964591  cice_runmod_mp_ic         213  CICE_RunMod.F90
```

Line 250 in icepack_atmo.F90 is: `` alz  = log(zlvl/zref)``

## Testing:

1. Have you tested the code changes? On what platforms?

Gaea was used for determining why the model was failing with the tested IC.

2. Have you run regression test in ufs-weather-model or ufs-s2s-model with code changes?

This fix will be updated in the associated ufs-weather-model PR. That PR will change baselines for all coupled tests.

## Dependent PRs:

- ufs-weather PR [#537](https://github.com/ufs-community/ufs-weather-model/pull/537)

### LINKED ISSUE #388: Revising oro data with full lakes only
We propose to ignore fractional lakes in the oro data until these small water bodies can be initialized realistically. This will change results in all tests that use oro data under "input-data-.../FV3_input_frac/".

### LINKED ISSUE #517: update inputdata for coupled benchmark tests; set nst_anl=T in benchmark v16 tests
## Description

Changes are required to input.nml and input data to match current P6 configuration.

## Solution

- Create new input-data-YYYYMMDD directory on all platforms with the updated ICs and orodata for the benchmark configuration date 20130401.
- Set nst_anl=T for benchmark v16 tests

Closes #314

### LINKED ISSUE #564: "bottom level" ATM exports for coupled model at initialization change depending on dt
## Description

It was noted in P6 that many failures at startup were solved by _increasing_ the dt from 225 to 300. This is contrary to what should be expected, that if the failure is caused by numerical instability, then larger dt should make the model _more_ likely to fail.

In fact, I think most, if not all, P6 failures at start-up are in fact caused by the negative zbot issue; they appear to be solved by changing dt because when dt is changed, the bottom-most values exported to the ICE model are different. And, in fact, the previous negative zbot value can now be positive when dt is changed.

It would not be expected that the initial "bottom" fields exported by the ATM are dependent on dt.

### To Reproduce:

The C96 dt tests described below can be run using the branch [feature/testdt](https://github.com/DeniseWorthen/ufs-weather-model) using:

./rt.sh -ek -l rt.test >output 2>&1 &

## Additional context

I began by testing the  S2S ``benchmark_v16`` regression test in debug mode. (I could not use the wave version because debug mode does not work for the S2SW.) The date is 2014-03-01 in the RT benchmark tests. I used the run directory created by this test and replaced the oro_data,sfc_data and gfs_data, ice and ocean inputs from the workflow input area to create a run directory for 2011-05-01. This was one of the P6 cases which failed on startup and ran when dt was increased to 300 (or when zbot was set to a minimum value of 1.0). I also inserted a coupler history write in the run sequence here:

```
# CMEPS warm run sequence
runSeq::
@1800
   MED med_phases_prep_ocn_avg
   MED -> OCN :remapMethod=redist
   OCN
   @225
     MED med_phases_prep_atm
     MED med_phases_prep_ice
     MED med_phases_history_write
```

This will write the fields imported by the mediator from the ATM prior to the ATM actually running. On the first pass through the run sequence, these will be the initial values from the ATM.

I ran this test at dt=225 and got an immediate failure due to a single point which contains a negative zbot, located on tile 6 at this location:

``n = 1 i,j 109  257  lat,lon = -63.97430929876248, 297.4105299440252  zbot = -1.957797348642381``

I then made the necessary changes in ``model_configure``, ``nems_configure`` and ``ice_in`` to run the exact same test at dt=300. In this case, there was **no longer any negative z-bot on tile6**. The only negative zbot was located on tile1 (which does not contain ice). Therefore, the model ran. 

In fact, 4 fields, all derived from the ``atmosphere_get_bottom_layer`` routine have different values at the initial import from the ATM with dt=300 vs dt=225:  ``atmImp_Sa_pbot``,``atmImp_Sa_shum``,``atmImp_Sa_tbot``,``atmImp_Sa_z``. All fields except these 4 fields were identical. The run directories for the v16 tests:

```
/scratch1/NCEPDEV/stmp2/Denise.Worthen/FV3_RT/tests_dt/v16test_dt225
/scratch1/NCEPDEV/stmp2/Denise.Worthen/FV3_RT/tests_dt/v16test_dt300
```

I then created a simpler test using the C96mx100L64 configuration using the proposed updated input. This test has previously been shown to fail because of a negative zbot exported to the ice (see [issue 388](https://github.com/ufs-community/ufs-weather-model/issues/388#issuecomment-828559457)). As expected, the model failed on the first timestep. I also ran a duplicate test (I created a ``cpld_debugfrac_dt`` test) with ``dt=720``. It also failed due to negative zbot but the number and locations of the negative zbot locations were completely different. Again, the four fields ``atmImp_Sa_pbot``,``atmImp_Sa_shum``,``atmImp_Sa_tbot``,``atmImp_Sa_z`` imported from ATM at initialization were different at the different dt values but all other fields imported from ATM were identical. Figures are shown below. The run directories for the c96mx100 dt tests:

```
/scratch1/NCEPDEV/stmp2/Denise.Worthen/FV3_RT/test_dts_c96/cpld_debugfrac
/scratch1/NCEPDEV/stmp2/Denise.Worthen/FV3_RT/test_dts_c96/cpld_debugfrac_dt
```

## Output

Difference in initial bottom temperature imported by the mediator from the ATM using dt 900 vs dt 720:
tile6 atmImp_Sa_tbot

Difference in initial bottom pressure imported by the mediator from the ATM using dt 900 vs dt 720:

tile6 atmImp_Sa_pbot

Difference in initial bottom height imported by the mediator from the ATM using dt 900 vs dt 720:

tile6 atmImp_Sa_z

### PR DESCRIPTION
**Pull request by DeniseWorthen**
# PR Checklist

- [X] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [x] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [X] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [X] If new or updated input data is required by this PR, it is clearly stated in the text of the PR.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

- Updates ``FV3_input_frac`` oro data and IC for full lakes only, consistent with input used for P6: Source directory: ``/scratch2/BMC/gsd-fv3-dev/FV3-MOM6-CICE5/inputdata_lakeint_p6/FV3_input_frac``
- Sets ``nst_anl=.true`` for benchmark v16 tests
- Switches to using a mesh for MOM6 in the cap (use_mommesh is set ``true``). See original issue [#398](https://github.com/ufs-community/ufs-weather-model/issues/398) 
- adds 2 additional suites: suite_FV3_GFS_v16_coupled_noahmp.xml, suite_FV3_GFS_v16_coupled_nsstNoahmp.xml

- A new ``input-data-20210518`` has been staged on on platforms. This was created by first rsync'ing the current ``input-data-20210428`` and then updating the files in the ``FV3_input_frac`` and the ``mesh.mx100.nc`` files in the ``CICE_FIX`` directories. 

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues must always be created before starting work on a PR branch!) 
- fixes #388 
- fixes #517
- fixes [#293](https://github.com/NOAA-EMC/fv3atm/issues/293)

Related issue #564. When ``na_init=1``, the bottom fields exported by ATM are expected to depend on dt. The fact that changing the timestep to 300s in P6 appeared to fix many of the cases which failed on startup is most likely due to chance. When dt was changed, the change in the bottom fields happened to remove the negative zbot which was sent to ICE.

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

Pre-commit testing on Gaea: a temporary baseline was created using the updated data and then verified against that baseline. All coupled and DATM tests passed. Standalone ATM tests were not run.

- [x] hera.intel
- [x] hera.gnu
- [x] orion.intel
- [x] cheyenne.intel 
- [x] cheyenne.gnu
- [x] gaea.intel 
- [x] jet.intel
- [x] wcoss_cray
- [x] wcoss_dell_p3

## Dependencies

GFDL_atmos_cubed_sphere PR [#102](https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/102)
fv3atm PR[#294](https://github.com/NOAA-EMC/fv3atm/pull/294)

co-author: @shansun6
co-author: @XiaqiongZhou-NOAA

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,7 +1,9 @@
 [submodule "FV3"]
   path = FV3
-  url = https://github.com/NOAA-EMC/fv3atm
-  branch = develop
+  #url = https://github.com/NOAA-EMC/fv3atm
+  #branch = develop
+  url = https://github.com/DeniseWorthen/fv3atm
+  branch = feature/noneg_zbot_export
```
Comment (junwang-noaa) [inline .gitmodules]: Is the fv3 branch updated? I see some changes in FV3/ccpp

Comment (DeniseWorthen) [reply .gitmodules]: I fixed it now.

**Comment by BrianCurtis-NOAA** [comment]
Machine: jet
Compiler: intel
Job: BL
Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/619583652/20210512180203/ufs-weather-model
Please manually delete: /lfs4/HFIP/h-nems/emc.nemspara/RT_RUNDIRS/emc.nemspara/FV3_RT/rt_174184
Test cpld_debug 017 failed in run_test failed 
Test cpld_debugfrac 018 failed in run_test failed 
Test cpld_control 001 failed in run_test failed 
Test cpld_controlfrac 002 failed in run_test failed 
Test cpld_satmedmf 003 failed in run_test failed 
Test cpld_ca 004 failed in run_test failed 
Test cpld_controlfrac_c192 006 failed in run_test failed 
Test cpld_bmarkfrac_v16 011 failed in run_test failed 
Test cpld_bmarkfrac_v16_nsst 012 failed in run_test failed 
Test cpld_control_c384 007 failed in run_test failed 
Test cpld_bmarkfrac 010 failed in run_test failed 
Test cpld_bmark 009 failed in run_test failed 
Test cpld_controlfrac_c384 008 failed in run_test failed 
Test cpld_control_wave 016 failed in run_test failed 
Test cpld_control_c192 005 failed in run_test failed 
Test cpld_bmarkfrac_wave_v16 015 failed in run_test failed 
Test cpld_bmarkfrac_wave 014 failed in run_test failed 
Test cpld_bmark_wave 013 failed in run_test failed 
Test fv3_stretched 034 failed in run_test failed 
Test fv3_stretched_nest 035 failed in run_test failed 
Test fv3_regional_quilt 038 failed in run_test failed 
Test fv3_regional_quilt_hafs 039 failed in run_test failed 
Test fv3_regional_quilt_RRTMGP 041 failed in run_test failed 
Test fv3_regional_control 036 failed in run_test failed 
Test fv3_regional_quilt_netcdf_parallel 040 failed in run_test failed 
Test fv3_csawmg 045 failed in run_test failed 
Test fv3_rrfs_v1alpha 052 failed in run_test failed 
Test fv3_gsd 051 failed in run_test failed 
Test fv3_thompson 053 failed in run_test failed 
Test fv3_thompson_no_aero 054 failed in run_test failed 
Test fv3_gfdlmprad_gwd 043 failed in run_test failed 
Test fv3_gfs_v16_RRTMGP 057 failed in run_test failed 
Test fv3_gfs_v16_stochy 056 failed in run_test failed 
Test fv3_gfs_v16 055 failed in run_test failed 
Test fv3_gfs_v16_RRTMGP_debug 068 failed in run_test failed 
Test fv3_gfs_v16_debug 067 failed in run_test failed 
Test fv3_gfsv16_csawmgt 059 failed in run_test failed 
Test fv3_gfsv16_csawmg 058 failed in run_test failed 
Test fv3_rrfs_v1beta_debug 076 failed in run_test failed 
Test fv3_gsd_diag3d_debug 073 failed in run_test failed 
Test fv3_thompson_debug 074 failed in run_test failed 
Test fv3_gsd_debug 072 failed in run_test failed 
Test fv3_regional_control_debug 069 failed in run_test failed 
Test fv3_stretched_nest_debug 071 failed in run_test failed 
Test fv3_rrfs_v1alpha_debug 077 failed in run_test failed 
Test fv3_thompson_no_aero_debug 075 failed in run_test failed 
Test fv3_gfsv16_ugwpv1 064 failed in run_test failed 
Test fv3_gfsv16_ugwpv1_warmstart 065 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson 062 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 063 failed in run_test failed 
Test fv3_gfs_v16_ras 066 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson_debug 079 failed in run_test failed 
Test fv3_gfsv16_ugwpv1_debug 080 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson_debug 078 failed in run_test failed 
Test fv3_gfs_v16_ras_debug 081 failed in run_test failed 
Test fv3_gocart_clm 060 failed in run_test failed 
Test fv3_gfs_v16_flake 061 failed in run_test failed 
Test datm_debug_cfsr 088 failed in run_test failed 
Test datm_cdeps_debug_cfsr 095 failed in run_test failed 
Test datm_control_cfsr 082 failed in run_test failed 
Test datm_bulk_cfsr 084 failed in run_test failed 
Test datm_mx025_cfsr 086 failed in run_test failed 
Test datm_mx025_gefs 087 failed in run_test failed 
Test datm_bulk_gefs 085 failed in run_test failed 
Test datm_control_gefs 083 failed in run_test failed 
Test datm_cdeps_control_cfsr 089 failed in run_test failed 
Test datm_cdeps_bulk_cfsr 091 failed in run_test failed 
Test datm_cdeps_mx025_cfsr 093 failed in run_test failed 
Test datm_cdeps_bulk_gefs 092 failed in run_test failed 
Test datm_cdeps_mx025_gefs 094 failed in run_test failed 
Test datm_cdeps_control_gefs 090 failed in run_test failed 
Test fv3_gfdlmprad 096 failed in run_test failed 
Test fv3_gfdlmprad_atmwav 097 failed in run_test failed 
Please make changes and add the following label back:
jet-intel-BL

**Comment by DeniseWorthen** [comment]
There appears to be a problem with the new input data directory. Some files are missing, some directories I can't ``ls``. Since the entire auto-BL seems to have failed, I will re-sync the input from Hera.

**Comment by DeniseWorthen** [comment]
There is not enough space to rsync the input directory from my local directory (where I transferred from Globus) to the baseline area. 

```
5.2G	./Minsuk.Ji
2.7T	./Bin.Li
106G	./Brian.Curtis
2.1G	./Mikyung.Lee
230G	./Denise.Worthen
1.4G	./Jun.Wang
8.8T	./emc.nemspara
24K	./Dusan.Jovic
12T	.
```

I've clean out (removed the rt_*)  in ``/mnt/lfs4/HFIP/h-nems/emc.nemspara/RT_RUNDIRS/emc.nemspara/FV3_RT`` and emc.nemspara is now down to 2.0T.

@binli2337 please delete what you can.

**Comment by MinsukJi-NOAA** [comment]
> There is not enough space to rsync the input directory from my local directory (where I transferred from Globus) to the baseline area.
> 
> ```
> 5.2G	./Minsuk.Ji
> 2.7T	./Bin.Li
> 106G	./Brian.Curtis
> 2.1G	./Mikyung.Lee
> 230G	./Denise.Worthen
> 1.4G	./Jun.Wang
> 8.8T	./emc.nemspara
> 24K	./Dusan.Jovic
> 12T	.
> ```
> 
> I've clean out (removed the rt_*) in `/mnt/lfs4/HFIP/h-nems/emc.nemspara/RT_RUNDIRS/emc.nemspara/FV3_RT` and emc.nemspara is now down to 2.0T.
> 
> @binli2337 please delete what you can.

I deleted my files.

**Comment by BrianCurtis-NOAA** [comment]
Machine: orion
Compiler: intel
Job: BL
Repo location: /work/noaa/nems/emc.nemspara/autort/pr/619583652/20210512124508/ufs-weather-model
Please manually delete: /work/noaa/stmp/bcurtis/stmp/bcurtis/FV3_RT/rt_55918
Repo location: /work/noaa/nems/emc.nemspara/autort/pr/619583652/20210512222251/ufs-weather-model
Please manually delete: /work/noaa/stmp/bcurtis/stmp/bcurtis/FV3_RT/rt_297177
Test cpld_restart_bmark 018 failed failed 
Test cpld_restart_bmark 018 failed in run_test failed 
Test cpld_restart_bmarkfrac_v16 023 failed failed 
Test cpld_restart_bmarkfrac_v16 023 failed in run_test failed 
Please make changes and add the following label back:
orion-intel-BL

**Review by DusanJovic-NOAA — APPROVED** [review]

**Review by JessicaMeixner-NOAA — APPROVED** [review]

**Comment by DeniseWorthen** [comment]
Still waiting for the two failed orion jobs. Since the restart jobs were the ones that failed, this will actually require 4 jobs to complete.

**Comment by BrianCurtis-NOAA** [comment]
At this point since we think it's just a system problem and not a code problem, we should just move on without those two tests?

**Comment by DeniseWorthen** [comment]
All RTs are complete and pass. My FV3 branch now points to updated dycore (3720aea). After fv3 is merged I will revert my submodules and run-ci.

**Comment by junwang-noaa** [comment]
FV3 is merged

**Review by climbfuji — APPROVED** [review]
fv3atm has is correct, changes look good

**Review by junwang-noaa — APPROVED** [review]

**Pull request merged**